serverManager/common/source/ISessionServerApp.h

```diff
@@ -43,7 +43,7 @@ class ISessionServerApp
     virtual bool isConnected() const = 0;
     virtual std::string getSessionManagementSocketName() const = 0;
     virtual firebolt::rialto::common::SessionServerState getInitialState() const = 0;
-    virtual int getAppId() const = 0;
+    virtual int getId() const = 0;
```

Comment:
minor: would this be clearer if named getServerId()?

Reply:
I will change it in next commit